lib/ansible/modules/system/systemd.py

```diff
@@ -360,17 +368,14 @@ def main():
         else:
             module.params['scope'] = 'system'
 
-    # if scope is 'system' or None, we can ignore as there is no extra switch.
-    # The other choices match the corresponding switch
-    if module.params['scope'] not in (None, 'system'):
-        systemctl += " --%s" % module.params['scope']
-
     if module.params['no_block']:
         systemctl += " --no-block"
 
     if module.params['force']:
         systemctl += " --force"
 
+    unit = module.params['name']
```

Comment:
This can be removed since the same line has been added recently: https://github.com/ansible/ansible/blob/83bf90e/lib/ansible/modules/system/systemd.py#L343